src/Microsoft.DotNet.Wpf/tests/UnitTests/PresentationCore.Tests/System/Windows/Input/MouseActionConverter.Tests.cs

```diff
@@ -0,0 +1,166 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+using System.ComponentModel.Design.Serialization;
+
+namespace System.Windows.Input
+{
+    public class MouseActionConverterTests
+    {
+        [Fact]
+        public void ConvertTo_ConvertFrom_ReturnsExpected()
```

Comment:
@h3xds1nz, is there any reason why we are not using `[Theory]` along with data here ? This seems a little shift from the general behavior.